main.go

```diff
@@ -115,6 +116,39 @@ type ObjRef struct {
 	Offloaded uint
 }
 
+// updateAutoretrieveIndex ticks every tickInterval and checks for new information to add to autoretrieve
+// If so, it updates the filecoin index with the new CIDs, saying they are present on autoretrieve
```

Comment:
`If so, it updates the filecoin indexer`?

Reply:
I'm assuming you want some clarification as to what that is? https://github.com/filecoin-project/storetheindex

Reply:
isn't `storetheindex` a filecoin indexer? https://github.com/filecoin-project/storetheindex#running-the-indexer-service